code/modules/mining/abandoned_crates.dm

```diff
@@ -171,7 +171,7 @@
 		if(108)
 			new /obj/item/circuitboard/machine/protolathe(src)
 		if(109)
-			var/chosen_circuit = pick(subtypesof(/obj/item/circuitboard) - /obj/item/circuitboard)
+			var/chosen_circuit = pick(subtypesof(/obj/item/circuitboard) - (/obj/item/circuitboard + /obj/item/circuitboard/computer + /obj/item/circuitboard/machine + /obj/item/circuitboard/machine/vending + /obj/item/circuitboard/machine/telecomms + /obj/item/circuitboard/computer/arcade + /obj/item/circuitboard/machine/shuttle + /obj/item/circuitboard/mecha + /obj/item/circuitboard/mecha/durand + /obj/item/circuitboard/mecha/gygax + /obj/item/circuitboard/mecha/honker + /obj/item/circuitboard/mecha/odysseus + /obj/item/circuitboard/mecha/phazon + /obj/item/circuitboard/mecha/ripley))
```

Comment:
Just snap this entire option entirely.

Reply:
entire option entirely


I gotta replace it anyway

Reply:
entire option entirely


Gotta replace it anyway